health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/configs/UICustomizations.js

```
@@ -15,85 +16,193 @@ const businessServiceMap = {};
-        case "Checklist Name":
-          return row?.additionalDetails?.name;
-          break;
-        case "Status":
-          const toggle = () => {
-            setIsActive(!isActive);
-          };
-          const switchText = isActive ? "Active" : "Inactive";
-          return (
-            <>
+        case "CHECKLIST_ROLE":
+          let str = row?.data?.role;
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)